nekoyume/Assets/_Scripts/UI/Widget/DccCollection.cs

```diff
@@ -52,6 +62,22 @@ protected override void Awake()
             {
                 Close(true);
             });
+            allButton.OnClick.Subscribe(tabButton =>
+            {
+                if (!tabButton.IsToggledOn)
+                {
+                    tabButton.SetToggledOn();
+                    petButton.SetToggledOff();
+                }
+            }).AddTo(gameObject);
+            petButton.OnClick.Subscribe(tabButton =>
+            {
+                if (!tabButton.IsToggledOn)
+                {
+                    tabButton.SetToggledOn();
+                    allButton.SetToggledOff();
+                }
+            }).AddTo(gameObject);
```

Comment:
I think this subscription should be handled in `OnEnabled` or `Show`, and dispose them in `OnDisabled`.